### PR TITLE
Solve example

### PR DESCRIPTION
**Pull request by frostedoyster**
Improve documentation for solve

----
:books: Documentation preview :books:: https://equistore--194.org.readthedocs.build/en/194/

### CODE REVIEW

File: docs/src/reference/python/operations/linear_algebra/solve.rst [hunk 1]
```diff
@@ -3,3 +3,59 @@ solve
...
+        ...             values = np.arange(0, 1).reshape(1, 1)
+        ...         ),
+        ...         values = y_regression
+        ...     )]
+        ... )
+
+        ... c = equistore.solve(X, y)
+
+        ... print(c.block().values)
+        """
```
Comment (Luthaf) [inline docs/src/reference/python/operations/linear_algebra/solve.rst:61]: So for doctest to do anything, you should also include the (expected) output of the print in the docstring.

Might also be worth adding some comments on what's happening around here

Comment (frostedoyster) [reply docs/src/reference/python/operations/linear_algebra/solve.rst]: @Luthaf I thought I'd fixed everything but I still get a blank page for solve... Do you know why?

Comment (agoscinski) [reply docs/src/reference/python/operations/linear_algebra/solve.rst]: It looks like a bug we recently fixed in https://github.com/lab-cosmo/equistore/pull/184
Can you try `git rebase master`?

Comment (frostedoyster) [reply docs/src/reference/python/operations/linear_algebra/solve.rst]: @agoscinski Thanks!

File: python/src/equistore/operations/solve.py [hunk 2]
```diff
@@ -22,6 +22,67 @@ def solve(X: TensorMap, Y: TensorMap) -> TensorMap:
...
+    ...         components = [],
+    ...         properties = Labels(
+    ...             names = ["property"],
+    ...             values = np.arange(0, 1).reshape(1, 1)
+    ...         ),
+    ...         values = y_regression
+    ...     )]
+    ... )
+    ...
+    >>> c = equistore.solve(X, y)
```
Comment (Luthaf) [inline python/src/equistore/operations/solve.py:81]: Maybe also print `c.block()` to show it's structure?

**Review by Luthaf — APPROVED** [review]

**Comment by Luthaf** [comment]
Thanks for the work @frostedoyster!

**Pull request merged**